lib/src/group_button_base.dart

```diff
@@ -263,12 +266,22 @@ class GroupButton extends StatelessWidget {
   /// Return int [index] of selected button
   final Function(int index)? onDisablePressed;
 
-  /// bool variable for switching between modes [ChackBox] and [Radio]
+  /// bool variable for switching between modes [CheckBox] and [Radio]
   ///
   /// if the [isRadio] = true, only one button can be selected
   /// if the [isRadio] = false, you can select several at once
   final bool isRadio;
 
+  /// bool variable for enable radio button to be deselected
+  ///
+  /// * if the [isRadio] = true :
+  /// - if the [isCanBeDeselect] = true , the selected radio button can be deselected
+  /// - if the [isCanBeDeselect] = false , the selected radio button can't be deselected
+  ///
+  /// * if the [isRadio] = false:
+  /// - [isCanBeDeselect] have no effect
+  final bool? isCanBeDeselect;
```

Comment:
In 4.3.0 version i add GroupButtonOptions model
Can you move this field in the class ?

Reply:
And you can change name of this field to enableDeselect
I think it's a little clearer

Reply:
> In 4.3.0 version i add GroupButtonOptions model Can you move this field in the class ?

but you said that GroupButtonOptions is for the UI settings, did you think this option that i added is can be UI setting ?